stories/accordion/index.tsx

```diff
@@ -62,28 +65,30 @@ const accordionWithKnobs = (
   </React.Fragment>
 );
 
-storiesOf('Accordion', module)
```

Comment:
If these changes were to try and wrangle 5.2 working, I think we should revert it to keep the diff relevant.

Reply:
no, this was to make the Knobs work. But it is also bugged in `master` (also alerts components) so I gave up, maybe we can get it fixed when we upgrade the `@storybook` and tis plugins

Reply:
reverted back to its initial state for now since it didnt work